src/SPC/util/SPCConfigUtil.php

```diff
@@ -93,7 +93,13 @@ private function getLibsString(array $libraries): string
         // patch: imagick (imagemagick wrapper) for linux needs -lgomp
         if (in_array('imagemagick', $libraries) && PHP_OS_FAMILY === 'Linux') {
             if (getenv('SPC_LIBC') === 'glibc') {
-                $short_name[] = '-l:libgomp.a -l:libgomp_nonshared.a';
+                $ld = f_exec('/usr/bin/gcc --print-file-name=libgomp.a', $output, $result_code);
```

Comment:
This is a bug as it'll report the system lication even if a different CC is used.

Reply:
It's just my test code for frankenphp fetching. I'll revert all of this things later.